rust/basic_bitcoin/dfx.json

```diff
@@ -1,6 +1,6 @@
 {
   "version": 1,
-  "dfx": "0.11.1",
+  "dfx": "0.12.0-beta.2",
```

Comment:
```suggestion
  "dfx": "0.12.0",
```

Reply:
```suggestion
```
I think with 0.12.0 available we can even remove this entirely